I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 125, in script
    $ equipment.equip(short_sword, "weapon")
  File "game/script.rpy", line 125, in <module>
    $ equipment.equip(short_sword, "weapon")
  File "game/inventory.rpy", line 97, in equip
    self.adjust_moveset(item)
AttributeError: 'Equipment' object has no attribute 'adjust_moveset'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 125, in script
    $ equipment.equip(short_sword, "weapon")
  File "D:\Git-Huh\Fragmented\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Git-Huh\Fragmented\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 125, in <module>
    $ equipment.equip(short_sword, "weapon")
  File "game/inventory.rpy", line 97, in equip
    self.adjust_moveset(item)
AttributeError: 'Equipment' object has no attribute 'adjust_moveset'

Windows-10-10.0.22635 AMD64
Ren'Py 8.1.3.23091805
Fragmented 1.0
Mon Nov 27 12:59:34 2023
